tests/nodeos_startup_catchup.py

```diff
@@ -142,6 +142,7 @@ def waitForNodeStarted(node):
     Print("Cycle through catchup scenarios")
     twoRounds=21*2*12
     for catchup_num in range(0, catchupCount):
+        time.sleep(5)
```

Comment:
likewise

Reply:
Yes. This PR morphed into get the tests passing PR.